slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java

```diff
@@ -107,13 +112,19 @@ public void importData(final InputStream input) throws IOException {
     }
   }
 
-  private void parseValidator(final Handle handle, final JsonNode node)
+  private void parseValidator(
+      final Handle handle, final JsonNode node, final Optional<List<String>> pubkeys)
       throws JsonProcessingException {
     if (node.isArray()) {
       throw new IllegalStateException("Element of 'data' was not an object");
     }
     final ObjectNode parentNode = (ObjectNode) node;
     final String pubKey = parentNode.required("pubkey").textValue();
+
+    if (pubkeys.isPresent() && !pubkeys.get().contains(pubKey)) {
+      LOG.info("Skipping data import for validator " + pubKey);
```

Comment:
I couldn't spot in the spec where we should skip rather than import the slashing data for existing keys - can you point it out please?

Reply:
this only skips if there's slashing data for keys that are *not* part of list of keystores to import. 

As discussed above though, we could also just import everything as lighthouse does.

Reply:
OK, I misunderstood. So to be clear, in the scenario where you are re-importing an existing key with some potentially new slashing data, the slashing data will be imported?

Reply:
> So to be clear, in the scenario where you are re-importing an existing keystore with some potentially new slashing data, the slashing data will be imported and the keystore will yield a 'duplicate' message?

yep, exactly.